### PR TITLE
Rs 2022 07 05.1

### PR DESCRIPTION
**Pull request by rschlaikjer**
Broken out because I didn't realize Ben wasn't going in chrono order. Can just sit here and get rebased on his changes later.

### CODE REVIEW

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 1]
```diff
@@ -113,13 +113,15 @@ The glacier reached the buildings on the surface and froze everything in an inst
...
+// Bro _hates_ what-ifs
+What if...
 }
 [sha:32ff5f0fae9059c3738c16f3eb14f9349eaa5616]{
 -- Page 524, Offset 10003.
 -- あのビルにまだ避難していない人間がいたのなら、その人間ごと、あらゆる熱を停止させた。
 // あらゆる熱を停止させた。This line is so weird but I decided to mostly keep it as is because it's shiki sort of unable to accept
-If there were people in that building who hadn't yet evacuated, every single one of them would have had their heat stolen.
+What if there were still people in those buildings who hadn't evacuated.
+ Any warmth those people might have had has now been forced to a halt.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:124]: "forced to a halt" feels a little more sudden than I think 停止させた lets on, at least in this context.
I had rendered this line as:
> If anyone left in that building hadn't yet evacuated, every single one of them would be cold now.

which is admittedly a bit of a strong localization

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 2]
```diff
@@ -113,13 +113,15 @@ The glacier reached the buildings on the surface and froze everything in an inst
 [sha:16de899845dcfa5dbcd6c52b0dce3c21254aad38]{
 -- Page 524, Offset 10002.
 -- ……もしもの話。
-...What if.
+// Bro _hates_ what-ifs
+What if...
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:117]: I had this rendered as "Just speaking hypothetically..." for the structure I had on the next line but if you want to keep もしもの話 for its THEMATIC importance, that makes sense

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 3]
```diff
@@ -159,7 +161,7 @@ But that glacier formed in an instant.
 [sha:a756d8ecb94b486da90c3b1e0f83c70b75e1d5e1]{
 -- Page 526, Offset 10011.
 -- 範囲は狭く、寒波は地を這うものだが、放たれてしまえば逃げる間もなく殺される。
-Its range is narrow, and the cold wave only crawls on the ground, but once it's released there would be no chance of escaping it.
+Its range is narrow, and the cold wave only crawls along the ground, but once it's released there would be no hope of escaping it.
 }
 [sha:41ed9c90a22fca2dd8fadcf78ba38586f400d38f]{
 -- Page 527, Offset 10012.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:167]: 10020:
"She's standing in that frigid air without a scratch on her...!"

I didn't even think algid was a real word (and neither does Firefox's spellcheck)

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 4]
```diff
@@ -245,17 +246,17 @@ Not with the sickly eyes from before, but with eyes that carefully observe their
...
 -- Page 534, Offset 10029.
 -- よくもおれから、ここまで<炎|ねつ>を奪ってくれた」
-How dare you take so much warmth from me."
+How dare you rob so much warmth from me."
 }
 [sha:66e837f6635ae869de7d2f521b5f4b8053160851]{
 -- Page 535, Offset 10030.
 -- 憎悪だけをこめた、刃物のような声。
-His voice was like a blade, filled with nothing but hatred.
+His voice is like a blade, filled with nothing but hatred.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:259]: I had rendered this as:
> His words drip with hatred, his voice like a blade.

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 5]
```diff
@@ -507,19 +510,17 @@ And then―――
...
 -- <彼|か>の領土はこの星の一部でありながら、“生きるに<能|あた>わず”と除外された生命圏。
-// I think this か is an inanimate reference here?
-// just a reading, dont worry, I got it. (P)
 Although this domain remains of the planet, all the same is it a biosphere foreclosed... in which none may live...
 }
 [sha:d5e217d9d0ade899bb9bf162e4b8b308f788e254]{
 -- Page 551, Offset 10081.
 -- いかに真祖の姫であろうと、この地獄に耐えきれるか……！」
-Though you may be the Princess of the True Ancestors... think not you can endure this hell...!"
+Though you may be the Princess of the True Ancestors... think not that you can endure this hell...!"
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:518]: prefer old

**Review by Avuxo — APPROVED** [review]
otsukare.

File: script/Arcueid/Day 5/05_00_ARC05_8.txt (hunk 1 above)
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:124]: maybe "any warmth they might've had would be gone now"

Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:124]: I kinda want to preserve the causative, 'cold now' and 'gone now' kinda drop that Vlov _did this_ to them. Looking at it again I kinda like what we did before with stolen. How about
> Any warmth they might've had has been snatched away. 

which is also a bit of a strong loc, but 'stopping their heat' is a bit dumb. If he's going for chuuni atoms stop vibrating imagery he should have spelled it out more.

File: script/Arcueid/Day 5/05_00_ARC05_8.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:167]: Flowers for algid gone

File: script/Arcueid/Day 5/05_00_ARC05_8.txt (hunk 1 above)
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:124]: that works for me

**Pull request merged**